### PR TITLE
WIP: New Send Screen

### PR DESCRIPTION
**Pull request by as-dr**
This is a functional WIP that updates the `SendPage.tsx` styling to match the Figma designs.

**TO DO**
1. Update font-sizes with global variables
1. Update colors with global variables
1. Update styling of `Send` button
1. Begin & complete styling of `Confirmation` page
1. Add Cancel/Back button
1. Append Cancel/Back and Send buttons to bottom of screen

Advocating that we merge this in, then branch out again later to complete the above to-do's.

### CODE REVIEW

File: basic-wallet/public/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="16" />
```
Comment (dmihal) [inline basic-wallet/public/index.html]: I didn't think this was a valid charset, can you explain this?

File: local-wallet/public/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-16" />
```
Comment (dmihal) [inline local-wallet/public/index.html]: Is there a reason that we're switching to UTF16?

Comment (as-dr) [reply local-wallet/public/index.html]: From @carlfairclough's  PR: `utf-16 html encoding so we can show utf-16 characters`. 

[Source](https://github.com/dmihal/maker-devcon-burner-wallet/pull/15)

In terms of the rationale behind that, I will have to leave it up to him.

**Comment by carlfairclough** [comment]
Switched back from utf-16 to utf-16. I didn't know about the &larr; character in HTML, so switched encoding to access that character

@dmihal @as-dr

**Comment by as-dr** [comment]
@dmihal Yours to merge! 🎉

File: basic-wallet/public/index.html (hunk 1 above)
Comment (dmihal) [reply basic-wallet/public/index.html]: This Charset still needs to be changed to utf-8, right?

**Review by dmihal — APPROVED** [review]

**Pull request merged**